testsuite/features/secondary/min_ansible_control_node.feature

```diff
@@ -12,31 +15,62 @@ Feature: Operate an Ansible control node in a normal minion
      Then I should see a "Ansible Control Node type has been applied." text
 
   Scenario: Apply highstate and check that Ansible is installed
-    Given I am on the Systems overview page of this "sle_minion"
-    When I follow "States" in the content area
-    And I click on "Apply Highstate"
-    And I wait until event "Apply highstate scheduled by admin" is completed
-    Then "ansible" should be installed on "sle_minion"
+     Given I am on the Systems overview page of this "sle_minion"
+     When I follow "States" in the content area
+     And I click on "Apply Highstate"
+     And I wait until event "Apply highstate scheduled by admin" is completed
+     Then "ansible" should be installed on "sle_minion"
 
   Scenario: The Ansible tab appears in the system overview page
      Given I am on the Systems overview page of this "sle_minion"
      When I follow "Ansible" in the content area
      Then I should see a "Ansible Control Node Configuration" text
 
-  Scenario: Pre-requisite: Deploy test playbooks and inventory file
-     Given I deploy testing playbooks and inventory files to "sle_minion" 
+  Scenario: Configure some inventory and playbooks path
+     Given I am on the Systems overview page of this "sle_minion"
+     When I follow "Ansible" in the content area
+     Then I should see a "Ansible Control Node Configuration" text
+     And I enter "/srv/playbooks/" as "new_playbook_path_input"
+     And I click on button with id "new_playbook_path_save"
```

Comment:
Could we use the text of the button instead of the ID?
It's more readable for a user following these steps.

Reply:
No, I think we cannot. 

Here we have two different "Save" buttons (with same text) in the same page, and the `<div>` where they are contained have no ids to distinguise between them. So I cannot reuse one of the existing step and had to implement this new one.